packages/gateway/src/metric/emitter.ts

```diff
@@ -0,0 +1,99 @@
+import { EventEmitter } from 'events';
+import { prefixLogger } from '@graphql-portal/logger';
+import { config } from '@graphql-portal/config';
+import { redis } from '../redis';
+import { MeshPubSub, ResolverData } from '@graphql-mesh/types';
+import { serializer, transformResolverData } from './utils';
+import MetricsChannels from './channels.enum';
+import PubSubEventsEnum from './pubsub-events.enum';
+
+const logger = prefixLogger('metric-emitter');
```

Comment:
Let's have a standard analytics-wide prefix, aka "analytics:{module name}", where _module name_ is optional. For this particular file it should be `prefixLogger('analytics:metric-emitter')`